app-emulation/sen/sen-0.5.1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5} )
```

Comment:
3.6 currently isn't supported by urwid (and urwidtrees), which are dependencies of app-emulation/sen, hence no 3.6. Once urwid has resolved all issues on 3.6 I wanted to make a new patch set enabling python3_6 for sen and the dependencies.